spec/models/user_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+describe User do
+  before do
+    User.delete_all
+  end
+
+  describe ".ordered" do
+    it "orders records by name ASC" do
+      beatrice = User.create!(name: "Beatrice", email: "beatrice@interexchange.org", password: "password")
+      adam = User.create!(name: "Adam", email: "adam@interexchange.org", password: "password")
+
+      expect(described_class.ordered).to eq([adam, beatrice])
+    end
+  end
```

Comment:
You should extend the examples here to include records with lower case names, to ensure the sorting works the same as in Book